### PR TITLE
Fix livesync ios when having both iOS and Android devices connected at the same time

### PR DESCRIPTION
**Pull request by Fatme**
Fixes https://github.com/telerik/mobile-cli-lib

Fixes the second part of this issue https://github.com/NativeScript/nativescript-cli/issues/654

### CODE REVIEW

File: commands/device/stop-application.ts [hunk 1]
```diff
@@ -13,11 +13,11 @@ export class StopApplicationOnDeviceCommand implements ICommand {
 		private $staticConfig: Config.IStaticConfig,
 		private $options: IOptions) { }
 
-	allowedParameters: ICommandParameter[] = [this.$stringParameter];
+	allowedParameters: ICommandParameter[] = [this.$stringParameter, this.$stringParameter];
 
 	public execute(args: string[]): IFuture<void> {
 		return (() => {
-			this.$devicesServices.initialize({ deviceId: this.$options.device, skipInferPlatform: true }).wait();
+			this.$devicesServices.initialize({ deviceId: this.$options.device, skipInferPlatform: true, platform: args[1] }).wait();
```
Comment (rosen-vladimirov) [inline commands/device/stop-application.ts]: why is this change required? this way you will not use the first passed argument, for example:
`$ appbuilder device stop android ios`
will omit android platform

Comment (rosen-vladimirov) [inline commands/device/stop-application.ts]: and in case you pass only one platform, it will be skipped, for example:
`$ appbuilder device stop android`

Comment (Fatme) [inline commands/device/stop-application.ts]: This command is not publicly exposed. We use it only as "dev" command because we need to exec stop application logic in separate process.

Comment (rosen-vladimirov) [inline commands/device/stop-application.ts]: Ok, I've got it now, the first argument is the app id, the second one is the platform :)

**Comment by rosen-vladimirov** [comment]
:+1:

File: commands/device/stop-application.ts (hunk 1 above)
Comment (Fatme) [inline commands/device/stop-application.ts]: This change is required because when livesyncing to ios device and android device is connected at the same time we're initializing and starting device discovery without platform. This triggers android discovery and thus throws an error. Actually initialing device service again is not correct (we've already initialized it at the begging of livesync command) and the change actually passed the specified platform to device service.
